packages/protocol/contracts/libs/LibAddress.sol

```diff
@@ -36,7 +37,31 @@ library LibAddress {
         if (!success) revert ETH_TRANSFER_FAILED();
     }
 
-    /// @dev Sends Ether to the specified address.
+    /// @dev Sends Ether to a thirdparty address. Note that the message call can only return up to
+    /// 64 bytes of data for security reasons.
+    /// @param _to The recipient address.
+    /// @param _amount The amount of Ether to send in wei.
+    function safeSendEther(address _to, uint256 _amount) internal {
+        safeSendEther(_to, _amount, gasleft());
+    }
+
+    /// @dev Sends Ether to the specified trusted address.
+    /// @param _to The recipient address.
+    /// @param _amount The amount of Ether to send in wei.
+    /// @param _gasLimit The max amount gas to pay for this transaction.
+    function sendEther(address _to, uint256 _amount, uint256 _gasLimit) internal {
+        // Check for zero-address transactions
+        if (_to == address(0)) revert ETH_TRANSFER_FAILED();
+
+        // Attempt to send Ether to the recipient address
+        (bool success,) = payable(_to).call{ value: _amount, gas: _gasLimit }("");
```

Comment:
Is it necessary to use `excessivelySafeCall` if the return data is never read like here? I would think no `returndatacopy` instruction is generated and so no attack is possible, though I haven't looked at the generated bytecode to confirm.

Reply:
According to [this issue](https://github.com/ethereum/solidity/issues/12306), the data is copied even if it's not used.